src/App.js

```diff
@@ -1,16 +1,28 @@
 import React, { Component } from 'react';
-import './App.css';
-import Movies from './components/Movies';
-import TopRated from './components/TopRated';
+import { Route, Switch, Link, NavLink } from "react-router-dom";
+import MovieDetails from './components/MovieDetails';
+import Home from './components/Home';
+// import Header from './components/Header';
+import About from './components/FooterLinks/About';
+import FAQ from './components/FooterLinks/FAQ';
+import ContactUs from './components/FooterLinks/ContactUs';
+import homeButton from './home.png';
 import { fetchMovieData } from './apiCalls'
+import './App.css';
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
+      logIn: this.logIn.bind(this),
+      logOut: this.logOut.bind(this),
+      leaveHome: this.leaveHome.bind(this),
+      goHome: this.goHome.bind(this),
       movies: [],
       isLoading: true,
-      errorMsg: null
+      errorMsg: null,
+      atHome: window.location.pathname === '/' ? true : false,
+      user: null
     }
   }
 
```

Comment:
We can take out this checkRes method below this line, seems to have been missed when the apiCalls got moved